src/webgl/context.js

```diff
@@ -23,21 +32,46 @@ and import 'luma.gl/headless' before importing 'luma.gl'.`;
 const ERR_HEADLESSGL_NOT_AVAILABLE =
 'Cannot create headless WebGL context, headlessGL not available';
 
-const ERR_HEADLESSGL_FAILED =
-'headlessGL failed to create headless WebGL context';
+const ERR_HEADLESSGL_FAILED = 'headlessGL failed to create headless WebGL context';
+
+export function isWebGLContext(gl) {
+  return gl && (gl instanceof WebGLRenderingContext ||
+    gl.ARRAY_BUFFER === GL_ARRAY_BUFFER);
```

Comment:
Is this the only way to detect if a debug context is WebGL 1 or WebGL 2? I'm sure it works, but make me feel uncomfortable